documentation/modules/evasion/windows/applocker_evasion_install_util.md

```diff
@@ -0,0 +1,9 @@
+## Intro
+
+This module is designed to evade solutions such as software restriction policies and Applocker.
+The main vector for this bypass is to use the trusted binary InstallUtil.exe in executing user supplied code.
+
+## Vulnerable Application
+
+This evasion will work on all versions of Windows that include .net versions 3.5 or greater (note: ensure the selected payload matches the target os architecture).
+
```

Comment:
Definitely going to need more here. ([ref](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation))

Sections to add:
* Options
* Scenarios